templates/phaser/packages/client/src/mud/getNetworkConfig.ts

```diff
@@ -1,22 +1,78 @@
+/* Network specific configuration for the client.
+ * By default connect to the anvil test network.
+ *
+ */
```

Comment:
```suggestion
/*
 * Network specific configuration for the client.
 * By default connect to the anvil test network.
 */
```
can we align these like all the other comments?